app/js/lib/validation_dispatchers/amount.js

```diff
@@ -17,13 +17,15 @@ var objectAssign = require( 'object-assign' ),
 	 * @return {ValidationDispatcher}
 	 */
 	createAmountValidationDispatcher = function ( validator,  initialValues ) {
-		var fieldNames = [ 'amount', 'paymentType' ];
+		var fieldNames = [ 'amount', 'paymentType' ],
+			fieldValuesIfNotSet = { amount: '0,00' };
```

Comment:
I'd rather name this `requiredFields` and also put `paymentType` in it as well, as we are not selecting a default payment type.